circuit_knitting/cutting/__init__.py

```diff
@@ -38,6 +38,23 @@
     instructions.CutWire
     instructions.Move
 
+Automatic Cut Finding
+~~~~~~~~~~~~~~~~~~~~~
+
+.. autosummary::
+    :toctree: ../stubs/
+    :nosignatures:
+
+    find_cuts
```

Comment:
If my [suggestion above](https://github.com/Qiskit-Extensions/circuit-knitting-toolbox/pull/520/files#r1544719292) is declined, this should actually be

```suggestion
    find_cuts.find_cuts
```

Reply:
Let's do this for right now, and we can continue discussing whether the settings should be importable alongside this in the `cutting` package